### ISSUE TITLE
Launcher fails to launch installations using custom command arguments

### ISSUE THREAD

**Issue by TresAbhi** [BedrockLauncher/BedrockLauncher#46]
**How to replicate**:
1. Open command prompt
2. `cd C:\Program Files (x86)\Minecraft Bedrock Launcher`
3. `./BedrockLauncher.exe --launch "YourUsernameHere" "YourInstallationNameHere"`

**Result:**
Launcher throws windows dialogue:
![](https://i.imgur.com/lws7Ug9.png)

**My exact steps:**
1. Open command prompt
2. `cd C:\Program Files (x86)\Minecraft Bedrock Launcher`
3. `./BedrockLauncher.exe --launch "CoolAbhi1290" "Latest Release (C)"`

**Context:**
Registered username: `CoolAbhi1290`
Installation name: `Latest Release (C)`
![](https://i.imgur.com/5GbJV8k.png)
![](https://i.imgur.com/yW3UA35.png)

**Log:**
```
Application Starting...
Application Pre-Initalization Finished!
Checking for updates
Current tag: 0.0.2.5
Latest tag: 0.0.2.5
System.Windows.Threading.DispatcherUnhandledExceptionEventArgs
```

**Comment by Rayth** [comment]
I'm surprised the ./ worked since windows should actually be .\ 

This shouldn't be the solution but just a note for using command prompt on windows.

**Comment by CoolAbhi1290** [comment]
@Rayth yeah, you can use forward slash and it still launches

**Comment by CarJem** [comment]
I've made attempts to fix this in 0.0.2.6 (Beta)
Verification would be appreciated.

**Comment by TresAbhi** [comment]
I'll test it in `2.0`
